### PR TITLE
Initial Portworx DC/OS Catalog

### PR DESCRIPTION
**Pull request by jsilberm**
Portworx provides scale-out, persistent data services for containers.

Our primary doc site (http://docs.portworx.com) provides instructions for how to easily deploy Portworx through Mesos and Marathon:  http://docs.portworx.com/run-with-mesosphere.html

### CODE REVIEW

**Comment by ryadav88** [comment]
@jsilberm thanks. Can I also ask you to write short getting started doc for this package so users don't have any issues installing and understanding how this works? you can repurpose your documentation for it. 

You can create a PR against https://github.com/dcos/examples and refer other examples listed there. These examples will finally go here: https://dcos.io/docs/1.8/usage/tutorials/. Please follow the template of other examples here.

This is also now part of the package submission guidelines: https://github.com/mesosphere/universe#submit-your-package

**Comment by jsilberm** [comment]
Hello --- Doc Examples PR submitted here: https://github.com/dcos/examples/pull/40

Thanks!

File: repo/packages/P/portworx/0/package.json [hunk 1]
```diff
@@ -0,0 +1,29 @@
+{
+  "packagingVersion": "3.0",
+  "minDcosReleaseVersion": "1.7",
+  "name": "portworx",
+  "version": "1.1",
```
Comment (ryadav88) [inline repo/packages/P/portworx/0/package.json]: `version` should be serviceVersion-packagingVersion 
Ex: if portworx version is 1.2.3 then `version` here will be `1.2.3-0.0.1`

File: repo/packages/P/portworx/0/package.json [hunk 2]
```diff
@@ -0,0 +1,29 @@
...
+  "scm": "https://github.com/portworx/px-dev",
+  "maintainer": "support@portworx.com",
+  "website": "http://www.portworx.com/",
+  "description": "Portworx PX provides scheduler integrated data services for containers.",
```
Comment (ryadav88) [inline repo/packages/P/portworx/0/package.json]: Please add more details in the description

File: repo/packages/P/portworx/0/package.json [hunk 3]
```diff
@@ -0,0 +1,29 @@
...
+    "portworx",
+    "storage",
+    "data",
+    "cloud",
+    "volume",
+    "s3",
+    "database"
+  ],
+  "preInstallNotes": "Please visit http://docs.portworx.com/run-with-mesosphere.html for detailed instructions",
+ "postInstallNotes": "",
```
Comment (ryadav88) [inline repo/packages/P/portworx/0/package.json]: add `postInstallNotes`

File: repo/packages/P/portworx/0/resource.json [hunk 4]
```diff
@@ -0,0 +1,14 @@
+{
+  "images": {
+    "icon-small": "https://raw.githubusercontent.com/portworx/px-dev/master/images/pwx-48.png",
+    "icon-medium": "https://raw.githubusercontent.com/portworx/px-dev/master/images/pwx-96.png",
+    "icon-large": "https://raw.githubusercontent.com/portworx/px-dev/master/images/pwx-256-1.png"
+  },
+  "assets": {
+    "container": {
+      "docker": {
+        "portworx-docker": "portworx/px-dev:latest"
```
Comment (ryadav88) [inline repo/packages/P/portworx/0/resource.json]: please use a  tagged version and not `latest`

File: repo/packages/P/portworx/0/package.json [hunk 5]
```diff
@@ -0,0 +1,29 @@
...
+  "tags": [
+    "portworx",
+    "storage",
+    "data",
+    "cloud",
+    "volume",
+    "s3",
+    "database"
+  ],
+  "preInstallNotes": "Please visit http://docs.portworx.com/run-with-mesosphere.html for detailed instructions",
```
Comment (ryadav88) [inline repo/packages/P/portworx/0/package.json]: add this line at the beginning of `preInstallNotes`: ```This DC/OS Service is currently EXPERIMENTAL. There may be bugs, incomplete features, incorrect documentation, or other discrepancies. Experimental packages should never be used in production!```

File: repo/packages/P/portworx/0/package.json (hunk 2 above)
Comment (ryadav88) [inline repo/packages/P/portworx/0/package.json]: add a link of portworx dcos/example at the end of `description`

File: repo/packages/P/portworx/0/package.json (hunk 1 above)
Comment (jsilberm) [inline repo/packages/P/portworx/0/package.json]: Fixed.

File: repo/packages/P/portworx/0/package.json (hunk 2 above)
Comment (jsilberm) [inline repo/packages/P/portworx/0/package.json]: Fixed and fixed.

File: repo/packages/P/portworx/0/package.json (hunk 5 above)
Comment (jsilberm) [inline repo/packages/P/portworx/0/package.json]: Fixed.

File: repo/packages/P/portworx/0/package.json (hunk 3 above)
Comment (jsilberm) [inline repo/packages/P/portworx/0/package.json]: Fixed.

File: repo/packages/P/portworx/0/resource.json (hunk 4 above)
Comment (jsilberm) [inline repo/packages/P/portworx/0/resource.json]: I've changed from "latest" to "1.1.2".   But we have a fairly responsive release cadence --- about once a month, not including critical updates.   Do you want/expect a corresponding stream of PR's?

**Comment by jsilberm** [comment]
@ryadav88 :  Can you please detail the service deployment problems?   Can you share the stdout/stderr logs?

**Comment by jsilberm** [comment]
@ryadav88 :  All checks have passed.  Sorry about the typo.

**Comment by jsilberm** [comment]
dcos_univ

@ryadav88 :  Just went through the example docs again, and it worked fine. 
Health checks work fine too.   (see screenshot)
Did you set up an etcd instance first?  Did you add an additional disk to the agent nodes?
Can you send me the output logs (stdout/stderr)?

**Pull request merged**